remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -69,6 +69,7 @@
 public class CategoriesImpl implements Categories
 {
     static final String INCLUDE_COUNT_PARAM = "count";
+    static final String INCLUDE_PATH_PARAM = "path";
```

Comment:
I think we could use same constant already present in `Nodes` interface here instead of duplicating this String literal.

Reply:
Right, refactored to use string from `Nodes`